.github/workflows/python-package.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow will install Python dependencies and run tests
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Python Testing
+
+on:
+  push:
+    branches: [ "main" ]
```

Comment:
I think you want to trigger tests also when you push to different branches than only main. Is that what you want?

Reply:
Default from github is this